app/views/document_lead_image/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= content_for :back_link, document_lead_image_path(@document) %>
+<% content_for :title, t("document_lead_image.edit.title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= image_tag(url_for(@image.thumbnail), id: "selected-image") %>
+    <%= form_tag(update_document_lead_image_path(@document, @image, next: params[:next]), method: :patch) do %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: t("document_lead_image.edit.form_labels.caption")
+        },
+        name: "caption",
+        value: @image.caption,
+      } %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: t("document_lead_image.edit.form_labels.alt_text")
+        },
+        name: "alt_text",
+        value: @image.alt_text,
+      } %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: t("document_lead_image.edit.form_labels.credit")
+        },
+        name: "credit",
+        value: @image.credit,
+      } %>
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Save image"
```

Comment:
Could this be 'Save and choose image', as it's actually doing two things?

Reply:
This has made me realise that a user should be able to edit the metadata of an image without selecting it as the lead image, so I will do some work to solve this.